include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -34,13 +34,18 @@ typedef struct _oe_sgx_enclave_image_info_t
 #define OE_SGX_FLAGS_KSS 0x0000000000000080ULL
 #define OE_SGX_SIGSTRUCT_SIZE 1808
 
+typedef struct _oe_sgx_enclave_flags_t
+{
+    uint32_t pf_gp_exceptions : 1;
```

Comment:
Based on the comments I see in the code, it seems like this should be called
```suggestion
    uint32_t capture_pf_gp_exceptions : 1;
```